instances/latest/terminologies/technique/superResolutionMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/superResolutionMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "Techniques in optical microscopy that allow images to have resolutions higher than those imposed by the diffraction limit, due to the diffraction of light [taken from [Wikipedia](https://en.wikipedia.org/wiki/Super-resolution_microscopy)].",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0739445",
+  "knowledgeSpaceLink": null,
+  "name": "super resolution microscopy",
+  "preferredOntologyIdentifier": null,
```

Comment:
Since you have an interlex ID, you should be able to fill the preferred ID in as well: http://uri.interlex.org/tgbugs/uris/indexes/ontologies/methods/218

Reply:
done